src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
Not sure what an "entrypoint" is supposed to be (outside of the Hydra context), actually. It's a Hydra thing. Perhaps we shouldn't try to shoehorn things...